### DISCUSSION TITLE
Terminal crash when compiling

### DISCUSSION THREAD

**Discussion by wesleyzzz** [idaholab/moose#25302] [Q&A Getting Started] [answered]
Hi all,

I am testing the MOOSE in Unbuntu 22 LTS in VMware Workstaion, following the instruction. It appears the terminal stops running
 when compiling a few steps as shown below, i.e., the compiling process gets stuck. Then several minuts later the terminal shuts down.

I updated the libmesh, but the problem persist.

Any comments?

(moose) wes@wes-virtual-machine:~$ cd ~/projects/moose/test
make -j 6
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/userobjects_Unity.C...
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/postprocessors_Unity.C...
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/ics_Unity.C...
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/geomsearch_Unity.C...
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/auxkernels_Unity.C...
Compiling C++ (in opt mode) /home/wes/projects/moose/framework/build/unity_src/reporters_Unity.C...

**Comment by GiudGiud** [comment, marked as answer]
Hello

you’re likely running out of memory. 

Try make -j1 for now. You can increase the number of processes if it worked

Guillaume

**Reply by wesleyzzz** [reply]
Thank you Guillaume. You are right, it was due to the low memory. I am able to compile and run.
Regards.
